src/runtime/testing.ts

```diff
@@ -79,23 +80,25 @@ export async function createTestContext(opts?: CreateTestContextOptions): Promis
   const layout = await Layout.create({ entrypointPath: opts?.entrypointPath })
   const pluginManifests = await Plugin.getUsedPlugins(layout)
   const randomPort = await getPort({ port: getPort.makeRange(4000, 6000) })
-  const app = require('../index').default as PrivateApp
```

Comment:
Not sure why it was like this before 🤔 

But seems to work without needing to be like this 😬 (🤞 no tests to be sure)